packages/features/ee/workflows/api/scheduleEmailReminders.ts

```diff
@@ -153,8 +193,10 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
   pageNumber = 0;
   const sendEmailPromises: Promise<any>[] = [];
 
+  const unscheduledReminders: PartialWorkflowReminder[] = [];
+
   while (true) {
-    const unscheduledReminders = await prisma.workflowReminder.findMany({
+    const newUnscheduledReminders = await prisma.workflowReminder.findMany({
```

Comment:
Same here. It makes more sense to have a workflowReminder getter with just different filters.